README.md

```diff
@@ -88,7 +88,7 @@ optional arguments:
 ### Decode
 
 ```
-usage: flask_session_cookie_manager.py decode [-h] [-s <string>] -c <string>
+usage: flask_session_cookie_manager{2,3}.py decode [-h] [-s <string>] -c <string>
```

Comment:
Yeah it was implicit that it requires either 2 or 3.

Reply:
I see what you mean, given the text under the `Usage` section states:
> Use flask_session_cookie_manager3.py with Python 3 and flask_session_cookie_manager2.py with Python 2.

However, this was the only example command which does **not** include the `{2,3}`.  It caught me out when using the library after a lazy copy-paste, so thought it might be useful to add, whilst making all usage examples consistent.